src/main/java/org/sagebionetworks/web/client/jsinterop/React.java

```diff
@@ -23,34 +23,27 @@ > ReactElement<T, P> createElement(
   public static native <T> T createRef();
 
   /**
-   * Similar to {@link #createElementWithSynapseContext} but only includes the theme. Any components rendered will NOT get the full Synapse context, including
-   * access token + auth state, experimental mode status, and time display settings.
+   * Wraps a component in SynapseContextProvider. Nearly all Synapse React Client components must be wrapped in this context, so this utility
+   * simplifies creating the wrapper.
+   *
+   * @param <P>
+   * @param componentType
+   * @return
    */
   @JsOverlay
   public static <
     T extends ReactComponentType<P>, P extends ReactComponentProps
-  > ReactElement<?, ?> createElementWithThemeContext(
```

Comment:
`createElementWithThemeContext` was a hacky workaround for cases where we could not use dependency injection. Now that we do not need to use DI, we can remove this function and replace it with `createElementWithSynapseContext`